pkg/master/master.go

```diff
@@ -47,9 +46,7 @@ import (
 	extensionsapiv1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
 	networkingapiv1 "k8s.io/kubernetes/pkg/apis/networking/v1"
 	policyapiv1beta1 "k8s.io/kubernetes/pkg/apis/policy/v1beta1"
-	rbacv1alpha1 "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
```

Comment:
actually, I just tried launching with the v1alpha1 group registered, but disabled by default, and v1alpha1 data was able to be read out of etcd just fine, so I'm cautiously ok with this. @deads2k, you concur?